compiler/examples/nba/transforms.js

```diff
@@ -22,7 +22,7 @@ var teamLogoTransform = new Transform("select * from teams;",
 
 var teamTimelineTransform = new Transform("select game_id, year, month, day, team1.abbr as home_team, team2.abbr as away_team, home_score, away_score, tier "
     + "from games, teams as team1, teams as team2 "
-    + "where games.home_team = team1.abbr and games.away_team = team2.abbr;",
+    + "where games.home_team = team1.abbr and games.away_team = team2.abbr and 1=1;", // 1=1 used for forcing recompute
```

Comment:
This looks a bit hacky to me - presumably user specification shouldn't be modified in order to force recompute. And I can't see why adding "1=1" will force recompute, can you explain a bit? 

Right now a less hacky way would be to run "node nba.js" while the backend is shutdown. Would that be easy enough to do? In any case, we should implement a more standard way to trigger recompute (maybe by passing arguments to "node nba.js")

Reply:
COMMENTED - sorry, I've now flagged this as a hack... as a hack, this is nice because it doesn't affect the core code before we've had a chance to discuss. yes absolutely, force-recompute deserves a longer offline conversation.

Here's some notes for future readers: the mechanism should work across data sources, i.e. no code required in the user-provided JS of course. Shutting down the backend is a bad solution - you never want to take things offline.

I looked into extending the REST protocol but currently, we just POST a JSON-encoded copy of the Project object, which makes it weird to include "now recompile this"... the annoying-to-code solution is to wrap this object with an "action" object... it seems easier to add a new REST endpoint or URL parameter, the classic is ?force=1.  All of this requires a discussion, hence the quick hack that's easily removed.

Reply:
Agree w/ all you said.  And looks to me that a new URL parameter should work. We can talk about it later. 

Btw I'm just curious: when in the booting process do you need to "recompute"? Before this recompute, have you already computed once?

Reply:
REMOVED - the "FORCE" env variable is now used to signal recompute.